yarn.lock

```diff
@@ -2,144 +2,152 @@
 # yarn lockfile v1
 
 
-"@babel/code-frame@^7.0.0", "@babel/code-frame@^7.5.5", "@babel/code-frame@^7.8.3":
```

Comment:
Did you manually update yarn.lock to cover this? I thought that with yarn, the preferred pattern was to update `package.json` with a `resolutions` property to enable yarn to handle this itself; was that not an option here?

Reply:
I did not manually update yarn. I just deleted yarn.lock & did yarn install again to update all packages. That fixed the issue. Alternatively i could have also added resolution in paqckage.json on npm to fix this issue. But, chose this option since we anyway try to be up to date with dependabot.